ee/tabby-webserver/src/service/auth.rs

```diff
@@ -709,12 +867,12 @@ mod tests {
         assert!(!users.page_info.has_next_page);
         assert!(users.page_info.has_previous_page);
 
-        let users = list_users(&conn, None, None, Some(3), None).await;
+        let users = list_users(&service, None, None, Some(3), None).await;
         assert!(!users.page_info.has_next_page);
         assert!(!users.page_info.has_previous_page);
 
         let users = list_users(
-            &conn,
+            &service,
             Some(all_users.edges[0].cursor.clone()),
             None,
             Some(2),
```

Comment:
This file is getting very long, do we want to split anything up?

Reply:
assuming it's not possible to split implementation of traits, seems no good way to split atm?

Reply:
There's a lot of utility functions and tests defined here that we could potentially move to another file

Reply:
sure - maybe a follow up PR with:
1. invitation.rs -> `/invitation/mod.rs`
2. splitting utilities into files under `/invitation/`